primer/brand
# Upgrade Mona Sans and Hubot Sans to 1.0.1 + Heading and Text component updates

## Summary

> [!IMPORTANT]  
> This PR contains breaking changes. 

Updates the libraries font files to use latest 1.0.1 release of Mona Sans and Hubot Sans. 

Hubot Sans is no longer dark-shipped, and both fonts are now exposed in `Heading` and `Text` components via a `font` prop.

**Review the visual diffs for before and after comparisons.**

## List of notable changes:




- **updated** Mona Sans font file, and replaced Alliance2 with Hubot Sans 
- **added** new `font` prop that defaults to Mona Sans
- **added** missing `extralight` weight option
- **added** new stories for viewing Mona Sans and Hubot Sans

## What should reviewers focus on?



- Check that the rendering of the fonts is correct
- Does anything look incorrect in the visual diffs?

## Steps to test:



1. Use the playground to experiment with new `font` options
1. [Review the Heading stories for Mona Sans and verify rendering appears correct](https://primer-4e9e924e15-26139705.drafts.github.io/storybook/?path=/story/components-heading--mona-sans)
1. [Review the Heading stories for Hubot Sans and verify rendering appears correct](https://primer-4e9e924e15-26139705.drafts.github.io/storybook/?path=/story/components-heading--hubot-sans)
1. [Review the Text stories for Mona Sans and verify rendering appears correct](https://primer-4e9e924e15-26139705.drafts.github.io/storybook/?path=/story/components-text--mona-sans)
1. [Review the Text stories for Hubot Sans and verify rendering appears correct](https://primer-4e9e924e15-26139705.drafts.github.io/storybook/?path=/story/components-text--hubot-sans)


## Supporting resources (related issues, external links, etc):

- [github/mona-sans](https://github.com/github/mona-sans)
- [github/hubot-sans](https://github.com/github/hubot-sans)

## Contributor checklist:


## Reviewer checklist:


## Screenshots:


Heading story for Mona Sans
![heading story](https://github.com/primer/brand/assets/13340707/4c724777-8aae-4192-81f9-eeb6a60a7422)


Text story for Mona Sans:

![image](https://github.com/primer/brand/assets/13340707/fe568d42-be83-4a84-b025-fdf65d84bdd2)
